test/utils.cpp

```diff
@@ -788,3 +788,23 @@ TEST_CASE(
 		unsetenv("BROWSER");
 	}
 }
+
+TEST_CASE(
+	"get_basename() returns basename from a URL if available, if not"
+	" then an empty string")
```

Comment:
Please add `"[util]"` as a second argument to the `TEST_CASE` macro; this lets us run tests for specific modules if need be.